### PR TITLE
Update approach and README

### PR DESCRIPTION
**Pull request by suprenant**

### CODE REVIEW

File: approach.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-3. Explore the feasibility of modeling it
-4. Develop a preliminary plan/recommendation for how to operationalize/build it out in order to meet NAVAIR’s goals
+2. Think critically about the focus areas and skillsets NAVAIR seeks to hire and retain
+3. Select a specific data set where we can quickly experiment and learn
+4. Develop possible insights on how to standardize, structure, and model the data in order to understand skill capabilities and gaps 
+5. Explore the feasibility of modeling this
+6. If modeling proves to be feasible, develop a preliminary recommendation for a path forward to build it out in order to meet NAVAIR’s goals. If not, recommend other next steps
+
 
+A review of this data,
```
Comment (tbaxter-18f) [inline approach.md]: Stray clause?

Comment (suprenant) [reply approach.md]: Thanks for the catch, @tbaxter-18f. Fixed, review again please?

File: approach.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 In order to do this, we will:
 1. Work collaboratively to review available data to determine the current state of the information
-2. Select a specific data set where we can quickly experiment and learn
-3. Explore the feasibility of modeling it
-4. Develop a preliminary plan/recommendation for how to operationalize/build it out in order to meet NAVAIR’s goals
+2. Think critically about the focus areas and skillsets NAVAIR seeks to hire and retain
+3. Select a specific data set where we can quickly experiment and learn
+4. Develop possible insights on how to standardize, structure, and model the data in order to understand skill capabilities and gaps 
+5. Explore the feasibility of modeling this
+6. If modeling proves to be feasible, develop a preliminary recommendation for a path forward to build it out in order to meet NAVAIR’s goals. If not, recommend other next steps
```
Comment (tbaxter-18f) [inline approach.md]: If bulleted items are full sentences -- or multi-sentence, in this case -- they should terminate with a period. If phrases, they don't. That's AP style, at least. At any rate, this one is awkward without the period, but if you change it you'd need to change the prior 5, too.

Comment (suprenant) [reply approach.md]: 100%, thank you. Fixed, take another look?

**Review by tbaxter-18f — APPROVED** [review]

**Pull request merged**